test/jdk/java/awt/FullScreen/FullScreenInsets/FullScreenInsets.java

```diff
@@ -96,60 +105,89 @@ public static void main(final String[] args) {
         try {
             r = new Robot(w.getGraphicsConfiguration().getDevice());
         } catch (AWTException e) {
             e.printStackTrace();
             passed = false;
             return;
         }
-        final BufferedImage bi = r.createScreenCapture(w.getBounds());
-        for (int y = 0; y < bi.getHeight(); y++) {
-            for (int x = 0; x < bi.getWidth(); x++) {
-                if (bi.getRGB(x, y) != color.getRGB()) {
-                    System.err.println(
-                            "Incorrect pixel at " + x + "x" + y + " : " +
-                            Integer.toHexString(bi.getRGB(x, y)) +
-                            " ,expected : " + Integer.toHexString(
-                                    color.getRGB()));
-                    passed = false;
-                    return;
-                }
+        final BufferedImage bimg = r.createScreenCapture(w.getBounds());
+        // vertical scan - at the far right side
+        int x = bimg.getWidth() - 1;
+        for (int y= 0; y < bimg.getHeight() - 1; y++) {
+            float[] actualRGB = new Color(bimg.getRGB(x, y))
+                    .getRGBColorComponents(null);
+            if (checkColor(actualRGB, expectedRGB)) {
+                System.out.println(
+                        "Vertical Scan: Incorrect pixel at " + x + "x" + y + " : " +
+                                Integer.toHexString(bimg.getRGB(x, y)) +
+                                " ,expected : " + Integer.toHexString(
+                                color.getRGB()));
+                passed = false;
+                break;
+            }
+        }
+        // horizontal scan - at the far bottom side
+        int y = bimg.getHeight() - 1;
+        for (x= 0; x < bimg.getWidth() - 1; x++) {
+            float[] actualRGB = new Color(bimg.getRGB(x, y))
+                    .getRGBColorComponents(null);
+            if (checkColor(actualRGB, expectedRGB)) {
+                System.out.println(
+                        "Horizontal Scan: Incorrect pixel at " + x + "x" + y + " : " +
+                                Integer.toHexString(bimg.getRGB(x, y)) +
+                                " ,expected : " + Integer.toHexString(
+                                color.getRGB()));
+                passed = false;
+                break;
             }
         }
+        if (!passed) {
+            ImageIO.write(bimg, "png", new File(filename));
+        }
+    }
+
+    private static boolean checkColor(float[] actualRGB, float[] expectedRGB) {
+        return (Math.abs(actualRGB[0] - expectedRGB[0]) > TOLERANCE
+                || Math.abs(actualRGB[1] - expectedRGB[1]) > TOLERANCE
+                || Math.abs(actualRGB[2] - expectedRGB[2]) > TOLERANCE);
     }
 
     private static void sleep() {
-        if(robot == null) {
+        if (robot == null) {
             try {
                 robot = new Robot();
-            }catch(AWTException ae) {
+            } catch(AWTException ae) {
```

Comment:
```suggestion
            } catch (AWTException ae) {
```

Reply:
There are a couple more lines where a space is missing on the left of a binary operator.